src/components/Post.js

```diff
@@ -151,18 +151,82 @@ export default class Post extends React.Component {
 const ExpandedPostDetails = props => {
   let {
     post, image, comments, commentsExpanded,
-    commentingDisabled, onCommentCreate, communities
+    commentingDisabled, onCommentCreate, communities, currentUser
   } = props
   let description = present(sanitize(post.description))
   let attachments = filter(post.media, m => m.type !== 'image')
   if (!comments) comments = []
 
+  var yeses = filter(post.responders, er => er.response === 'yes')
```

Comment:
@levity - there's so much code specifically for the event responses, I'm tempted to pull it out into it's own component. What are your thoughts?